### DISCUSSION TITLE
Hosting solution ?

### DISCUSSION THREAD

**Discussion by KizeRemi** [teslamate-org/teslamate#2603] [Q&A]
Hello,

I am looking for a solution to see a TeslaMate hosted? 
I don't want to host it at home, let alone maintain it on a server :( 

Do you know of a solution that makes it easy to do so? Even in SaaS mode?

Thanks

**Comment by ktsenvolt** [comment]
https://www.youtube.com/watch?v=1_9msvTEkpE&ab_channel=TeslaK%C3%A4gi

**Comment by parkr** [comment]
Much of the point of teslamate is to self-host. If you don't care to self-host, I would recommend signing up for TeslaFi instead. There are others too. Here's a somewhat recent post on hosted solutions: https://teslamotorsclub.com/tmc/threads/teslafi-vs-tezlab-vs-tessie-vs-dashboard-for-tesla-vs-scan-my-tesla-vs-teslalogger.252503/

**Comment by flemzord** [comment]
I am developing a hosting solution for TeslaMate: https://MyTeslaMate.com 
I think you should like it @KizeRemi

**Comment by bahree** [comment]
Whilst not a SaaS option, I did call out step-by-step direction on how to deploy this in a VM on Azure as one option - https://github.com/adriankumpf/teslamate/discussions/2253
